roles/kubespray-defaults/defaults/main.yaml

```diff
@@ -40,7 +43,7 @@ cluster_name: cluster.local
 # Subdomains of DNS domain to be resolved via /etc/resolv.conf for hostnet pods
 ndots: 2
 # Can be dnsmasq_kubedns, kubedns, manual or none
-dns_mode: kubedns
+dns_mode: coredns
```

Comment:
switching the default dns provider means we should clean up old kubedns. it will likely fail on upgrade because there is already a service created listening at that same cluster ip